### DISCUSSION TITLE
Vercel not serving js file | Getting 404 error

### DISCUSSION THREAD

**Discussion by kernelguardian** [vercel/community#3942] [Help] [answered]
### Summary

Vercel is not returning the js file for some reason.

<img width="1251" alt="image" src="https://github.com/vercel/community/assets/20723393/34f95385-a7a5-498e-9637-78d6337eecf9">

Here is the response header for the 404'd resource
<img width="552" alt="image" src="https://github.com/vercel/community/assets/20723393/f771786b-654c-4c0f-8207-bdd71f340f72">

I deployed the same react web app with netlify and it works fine.

### Steps to Reproduce

https://app.presenter.kernelguardian.com

**Comment by amyegan** [comment]
Hey @kernelguardian. I don't know enough about your code to be able to say what's going wrong. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share?

**Reply by kernelguardian** [reply, marked as answer]
Hey @amyegan I think it was an issue with the local browser cache. I cleared all the browser data and it worked after a couple of reload.
Thanks for the help.
